lib/matplotlib/axes/_base.py

```diff
@@ -1566,6 +1566,11 @@ def set_prop_cycle(self, *args, **kwargs):
         self._get_patches_for_fill.set_prop_cycle(prop_cycle)
 
     def get_aspect(self):
+        """
+        Return the aspect of the axis scaling.
```

Comment:
axis or axes?

Reply:
I was following the `set_aspect` docs. When looking at it again, `axes` makes more sense. Also changed to "aspect *ratio*". I assume "the aspect of the scaling" is a bit slang and may not be clear.

Reply:
I think it's axis?  (in the sense of x and y axis)
We should try to use "axis" when referring to x or y axis, even if it's in a plural context, because axes is just too easily confused with the unrelated Axes class.  (I also write "axises" in commit messages sometimes, but I guess that's may just read weird for casual users...)

Reply:
IMHO, one can justify "axis", "axes" and "Axes", and I don't have a strong preference for any of them. 
But I don't want to switch back and forth. So please, anybody who has opinions here, come to an agreement and I'll write that.

Technically, aspect ratio seems to be used on a single entity with two parts (or aspects), e.g. "the aspect ratio of an image".  Additional complication: here, we don't have a ratio of sizes, but a ratio of scalings (not even sure if that's commonly also identified as an aspect ratio, but doesn't matter in the end. It's part of our API and we won't change that). One could say that we have one overall scaling with an x-scaling and an y-scaling part. Whether we identify this scaling as a scaling applying to the "Axes", or applying to multiple axises, i.e. "axes" or using the singular "axis" to qualify the scaling as "axis scailing" seens quite the same to me.

On a side note: I agree that we should internally use "axises" as plural of "axis", but we should not make that public terminology.

Reply:
You can't have an aspect without two axises and an axes encapsulates the two axises.  I think it is the axes that has an aspect ratio.